integration_tests/CMakeLists.txt

```diff
@@ -169,6 +169,8 @@ RUN(NAME test_unary_minus    LABELS cpython llvm)
 RUN(NAME test_unary_plus     LABELS cpython llvm)
 RUN(NAME test_issue_518      LABELS cpython llvm)
 RUN(NAME structs_01          LABELS cpython llvm)
+RUN(NAME structs_02          LABELS llvm)
+RUN(NAME structs_03          LABELS llvm)
```

Comment:
As far as I can figure out, `structs_02.py` isn't possible with CPython because `ccallable` isn't supported there. For `struct_03.py` I get the following error,

```bash
Traceback (most recent call last):
  File "/Users/czgdp1807/lpython_project/lpython/integration_tests/structs_03.py", line 8, in <module>
    def f(pa: Pointer[A]):
TypeError: 'type' object is not subscriptable
```

@certik Do you know how to fix the above?

Reply:
CPython can't use `ccallable`, at least in the near future. The only way to make it work is by wrapping LPython itself into Python and calling it to take care of this. Which might not be a bad idea, but it's heavyweight and we should do that later. For now we won't test `ccallable` using cpython.

Regarding the above error, I think `Pointer[A]` does not work in CPython yet. That one we should get working.